loadtest/control/actions.go

```diff
@@ -217,6 +217,29 @@ func CreatePost(u user.User) UserActionResponse {
 	return UserActionResponse{Info: fmt.Sprintf("post created, id %v", postId)}
 }
 
+// EditPost updates a post.
+func EditPost(u user.User) UserActionResponse {
+	channel, err := u.Store().CurrentChannel()
+	if err != nil {
+		return UserActionResponse{Err: NewUserError(err)}
+	}
```

Comment:
Currently we don't really use `CurrentChannel` in `SimpleController`. And maybe we don't have too (see my comment above).
My suggestion was to add the random picking function (which we already had apparently) so that it could be used in the more realistic version.
Here I suggest we get a channel the user is member of (using our latest bitmask technology) and ask for a post for that one.